<ui version="4.0" >
 <class>ImprimirSobreClienteClass</class>
 <widget class="QWidget" name="ImprimirSobreClienteClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ImprimirSobreCliente</string>
  </property>
  <widget class="QWidget" name="gridLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="2" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Al</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Del</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QLineEdit" name="lineEdit" />
    </item>
    <item row="0" column="3" >
     <widget class="QLineEdit" name="lineEdit_2" />
    </item>
    <item row="1" column="0" colspan="4" >
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton" >
        <property name="text" >
         <string>Aceptar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2" >
        <property name="text" >
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ImprimirSobreClienteClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>414</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel" >
     <x>462</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
